### DISCUSSION TITLE
nRF24 wird nach Update nicht mehr erkannt

### DISCUSSION THREAD

**Discussion by haujau** [tbnobody/OpenDTU#1223] [Q&A]
Moin zusammen,

ich nutze openDTU seit November 2022 und habe mich jetzt (leider) entschlossen, ein Update auf die aktuelle Version durchzuführen. Bis zum Update funtze alles mit meinem HM-600 einwandfrei.
Nachdem ich gelesen hatte, dass ich vorher die ESP32 Partition ändern muss, habe ich via Python esptool das Flash gelöscht und den ESP32 via USB mit dem neu kompilierten Image geflasht.
Ich kann soweit alles konfigurieren und einrichten, nur die Hardware Einstellungen sehen etwas komisch aus. Ich kann dort nur das Profil "Standard" auswählen aber keine weiteren Einstellungen vornehmen. Unter "Nrf24" steht für alle Leitungen unter "Ausgewählt" nichts und unter "Aktiv" jeweils "-1".
Im Menü "System Informationen" ist der Staus für nRF24 "nicht konfiguriert". Als PIO Umgebung wird "generic_esp32" angegeben, als Chip-Modell "ESP32-D0WDQ6".
Ich vermute, dass ich in der platformio.ini das Portmapping anpassen muss. Obwohl das Mapping, was unter "env:generic" steht mit meiner Verkabelung (die bis zum Update ja funzte) übereinstimmt.
Hat jemand eine Idee, was ich anpassen muss, damit das nRF24 Modul wieder erkannt wird?

Gruß
Andreas

**Comment by tbnobody** [comment]
Alle neu geflashten ESPs benötigen ein Device Profile.
Aktuell kennt dein Gerät noch keine Pin Belegung. Dafür musst du ein passendes Device Profile einspielen. https://github.com/tbnobody/OpenDTU/wiki/Device-Profiles

Das hier entspricht der vorherigen Pin Zuordnung:
https://github.com/tbnobody/OpenDTU/blob/master/docs/DeviceProfiles/nodemcu_esp32.json

**Comment by haujau** [comment]
Vielen Dank für die schnelle Antwort!
Das hatte wohl überlesen... :(
Sorry!
Läuft jetzt wieder.
 
am Montag, 7. August 2023 um 18:09 schrieben Sie:
